src/Compilers/CSharp/Portable/Symbols/NamespaceSymbol.cs

```diff
@@ -245,43 +246,23 @@ internal NamedTypeSymbol ImplicitType
         /// Symbol for the most nested namespace, if found. Nothing 
         /// if namespace or any part of it can not be found.
         /// </returns>
-        internal NamespaceSymbol LookupNestedNamespace(ImmutableArray<string> names)
+        internal NamespaceSymbol LookupNestedNamespace(ImmutableArray<ReadOnlyMemory<char>> names)
```

Comment:
i couldn't find a cheap way to avoid the allocs.  i attempted to use TemporaryArray, but then got analyzer errors about exposing that type off of MetadataTypeName.